src/components/ECharts/ChainChart/index.tsx

```diff
@@ -126,115 +157,139 @@ export default function AreaChart({
-					}
+						color
+					},
+					areaStyle: areaColor
 				})
 				data?.feesChart.forEach(([date, value]) => {
 					series[series.length - 1].data.push([getUtcDateObject(date), value])
 				})
 			}
 
 			if (route.revenue === 'true' && data?.feesChart) {
+				const color = getColor(isCompare) || colors.revenue
+				const areaColor = getAreaColor(color, isThemeDark)
 				series.push({
 					name: namePrefix + 'Revenue',
 					chartId: 'Revenue',
-					type: 'bar',
+					symbol: 'none',
+					type: groupBy === 'cumulative' ? 'line' : 'bar',
 					data: [],
 					yAxisIndex: 3,
 					itemStyle: {
-						color: getColor(isCompare) || colors.revenue
-					}
+						color
+					},
+					areaStyle: areaColor
 				})
 				data?.feesChart.forEach(([date, _, value]) => {
 					series[series.length - 1].data.push([getUtcDateObject(date), value])
 				})
 			}
 
 			if (route.price === 'true' && data?.priceData && denomination === 'USD') {
+				const color = getColor(isCompare) || colors.price
+				const areaColor = getAreaColor(color, isThemeDark)
 				series.push({
 					name: namePrefix + 'Price',
 					chartId: 'Price',
 					symbol: 'none',
 					type: 'line',
 					data: [],
+
 					yAxisIndex: 4,
 					itemStyle: {
-						color: getColor(isCompare) || colors.price
-					}
+						color
+					},
+					areaStyle: areaColor
 				})
 				data?.priceData.forEach(([date, value]) => {
 					if (Number(date) > Number(data?.globalChart[0][0]))
 						series[series.length - 1].data.push([getUtcDateObject(date), value])
 				})
 			}
 
-			if (route.addresses === 'true' && data?.usersData?.length > 0) {
+			if (route.users === 'true' && data?.usersData?.length > 0) {
+				const color = getColor(isCompare) || colors.returningUsers
+				const areaColor = getAreaColor(color, isThemeDark)
 				series.push({
-					name: namePrefix + 'Returning Addresses',
-					chartId: 'Addresses',
-					stack: 'Addresses',
-					type: 'bar',
+					name: namePrefix + 'Returning Users',
+					chartId: 'Users',
+					stack: 'Users',
+					symbol: 'none',
+					type: groupBy === 'cumulative' ? 'line' : 'bar',
 					data: [],
 					yAxisIndex: 5,
 					itemStyle: {
-						color: getColor(isCompare) || colors.returningUsers
-					}
+						color
+					},
+					areaStyle: areaColor
 				})
 				data?.usersData.forEach(([date, value, value2]) => {
 					series[series.length - 1].data.push([getUtcDateObject(date), (value ?? 0) - (value2 ?? 0)])
 				})
-				if (data?.usersData?.[0]?.[2] !== undefined) {
-					series.push({
-						name: namePrefix + 'New Addresses',
-						chartId: 'Addresses',
-						stack: 'Addresses',
-						type: 'bar',
-						data: [],
-						yAxisIndex: 5,
-						itemStyle: {
-							color: getColor(isCompare) || colors.newUsers
-						}
-					})
-					data?.usersData.forEach(([date, value, value2]) => {
-						series[series.length - 1].data.push([getUtcDateObject(date), value2 ?? 0])
-					})
-				}
+				series.push({
```

Comment:
hmm is the change assuming that we'll have new user info for all protocols for which we have user info?